{"name": "Негативная проверка доверенного номера не найденного клиента", "status": "passed", "description": "Проверка неудачного поиска клиента по доверенному номеру в формате 10 символов", "steps": [{"name": "Проверка сообщения об ошибке и кода ответа", "status": "passed", "start": 1715340917371, "stop": 1715340917371}], "parameters": [{"name": "proofNum", "value": "'0553565311'"}], "start": 1715340917183, "stop": 1715340917372, "uuid": "739fd16f-ef75-43ca-aefc-7a8c68a53db0", "historyId": "c36b2762d5fbe2fecf895e31e1b0d87d", "testCaseId": "1a34d257530ec8961396b7a17e6623ea", "fullName": "207_service_test.TestProofNumSuite#test_get_info_client_10_letter_fiz_in_proofNum_get_success_response", "labels": [{"name": "subSuite", "value": "Проверки по доверенному номеру"}, {"name": "suite", "value": "Получение информации по клиенту (207 сервис)"}, {"name": "host", "value": "PC-411-140322D"}, {"name": "thread", "value": "28604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "207_service_test"}]}